api/src/main/java/org/openmrs/Order.java

```diff
@@ -843,4 +871,21 @@ public Double getSortWeight() {
 	public void setSortWeight(Double sortWeight) {
 		this.sortWeight = sortWeight;
 	}
+	
```

Comment:
Actually the @ since should be on these getters/setters since they are public and not the fields which are private

Reply:
Looks like they are on these setters, am I missing something?  I'm fine if it's one the private fields as well for redundancy.

Reply:
I was just saying they are redundant on a field because they are private, may be they can be useful for developers new to the project to quickly know when the properties were introduced by just glancing at the fields.